examples/model_selection/plot_precision_recall.py

```diff
@@ -140,10 +140,11 @@
 
 precision, recall, _ = precision_recall_curve(y_test, y_score)
 
+# no step argument for plt.fill_between for matplotlib < 1.5
+step_kwargs = {'step': 'post'} if hasattr(plt.cm, 'viridis') else {}
```

Comment:
Testing for the `viridis` colormap as an indication of the matplotlib version seems a bit convoluted.. I would be less surprised by a `if 'step' in inspect.signature(plt.fill_between)` but that's PY3 only, unfortunately. Is it too inconvinient to just check for the mpl version?

Reply:
Debatable I agree. I used sklearn.externals.signature instead.